src/App.js

```diff
@@ -4,6 +4,8 @@ import { Routes, Route } from "react-router-dom";
 import ProductListingPage from "./pages/ProductListingPage";
 import Navbar from "./components/NavBar/Navbar";
 import Footer from "./components/Footer/Footer";
+import Signup from "./components/Auth/Signup";
```

Comment:
All these components are coming from the components folder so you could create an `index.js` file in components folder and then export all the components from that index.js file, so that while importing you can just do that in a single line.

e.g.
```jsx
import { Navbar, Footer, Signup } from "./components";
```